scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+function reportFailure() {
+  if [ -n "${SLACK_WEBHOOK}" ] && [ -n "${BUILD_LABEL}" ]
+  then
+    BRANCH_NAME=$(git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\1/')
+    SLACK_MESSAGE=":failed_build: ${BUILD_LABEL} ${BROWSER} ${BVER} on branch ${BRANCH_NAME}"
+    echo "Reporting to slack: ${SLACK_MESSAGE}"
+    curl -X POST -H 'Content-type: application/json' --data '{"text": '\""$SLACK_MESSAGE"\"'}' $SLACK_WEBHOOK
+  fi
```

Comment:
Tangent: `fi` always makes me sad to look at 😂

Reply:
Is it required? I remove if it's not required.

Reply:
I unfortunately believe it is required. I just don't like the syntax of the language LOL.